notebook/static/tree/js/notebooklist.js

```diff
@@ -815,14 +814,14 @@ define([
         //#issue 3961, update the checkbox aria-label when it changed
         if(selected.length>=1){
           if($('#select-all').prop("checked")){
-            $('#button-select-all').attr("aria-label","Selected All "+ selected.length+" items");
+            $('#button-select-all').attr("aria-label", i18n.msg._("Selected All "+ selected.length+" items"));
           }
           else{
-            $('#button-select-all').attr("aria-label","Selected, "+ selected.length+" items");
+            $('#button-select-all').attr("aria-label", i18n.msg._("Selected, "+ selected.length+" items"));
```

Comment:
I think it needs to have a string literal inside the `_()` so that the translation tools can pick it up as something that needs to be translated (they do some kind of simple static analysis). I'm not 100% sure how that works with numbers - can you hunt around the codebase to see if we do that anywhere else? Or look for info online.

Reply:
Based on research through the project/online I think this I have added a way that takes into account non string literals such as numbers